invoices/views.py

```diff
@@ -587,8 +635,11 @@ def invoice_page(request, invoice_id, **_):
 
 
     previous_invoices = []
+    recent_weekly_report = None
     if invoice.project_m:
         previous_invoices = Invoice.objects.filter(project_m=invoice.project_m)
+        weekly_reports = WeeklyReport.objects.filter(project_m=invoice.project_m, client=invoice.client)
+        recent_weekly_report = weekly_reports[0]
```

Comment:
What happens here if there is no weekly reports for some reason? Does `invoice.project_m` guarantee there is always weekly reports available?